test.disabled/ejabberd_tests/tsung/mam_con.xml

```diff
@@ -58,7 +58,7 @@
       <!-- Returns a random message id of 24 hours age -->
       <![CDATA[
         fun() ->
-            Now = now_to_microseconds(now()),
+            Now = now_to_microseconds(os:timestamp()),
```

Comment:
😆 I completely forgot we still have some tsung scenarios in tests. I wonder if we could remove them now. I don't remember when they were used, especially now when we have amoc and tide.